### ISSUE TITLE
Bootstrap loaded before php settings are processed

### ISSUE THREAD

**Issue by bendavies** [paratestphp/paratest#348]
Given a `phpunit.xml` like this:

```xml
<?xml version="1.0" encoding="UTF-8"?>

<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
         backupGlobals="false"
         colors="true"
         bootstrap="bootstrap.php"
>
    <php>
        <env name="APP_ENV" value="test" />
    </php>

    <testsuites>
        <testsuite name="Project Test Suite">
            <directory>tests</directory>
        </testsuite>
    </testsuites>
</phpunit>
```

PHPUnit will process the `<php>` block first, i.e set the `APP_ENV` env var, and then run `bootstrap.php`.

Paratest does the opposite: runs `bootstrap.php` then set the `APP_ENV` env var.

This means that if the bootstrap relies in `APP_ENV`, `phpunit` will work, while `paratest` will not.

**Comment by moehrenzahn** [comment]
I can confirm this issue exists in ParaTest 2.2.0.

**Comment by jccpdev** [comment]
Also seems to be broken in 2.1

**Comment by Slamdunk** [comment]
Closed in #495
